R/model_functions.R

```diff
@@ -250,7 +250,7 @@ predict.comp_bayes_lm <- function(object, newdata, ...) {
 #'
 #' run_cv(
 #'   focal_vs_comp_ex,
-#'   comp_dist = 1,
+#'   comp_dist = c(1, 2),
 #'   blocks = blocks_ex
 #' )
 run_cv <- function(focal_vs_comp, comp_dist, blocks, prior_param = NULL, run_shuffle = FALSE) {
```

Comment:
The `run_cv()` uses a static value of `comp_dist` to compute the `focal_vs_comp` data frame; rather it should change per each value of the `comp_dist` argument vector.

At first glance it seems that the function should be of rough form:
```r
run_cv_output <- 
  run_cv(
    growth_df = ,
    comp_dist = seq(from = 1, to = 15, by = 0.5),
    blocks = blocks_scbi,
    id = "stemID"
  )
```
Note the arguments match those of the `create_focal_vs_comp()` function so we can prolly do an `@inheritsParams` for the roxygen. 

Then internally `run_cv()` computes a new `focal_vs_comp` data frame for each value of `comp_dist`